docs/en/stack/ml/anomaly-detection/anomaly-detection-scale.asciidoc

```diff
@@ -0,0 +1,228 @@
+
+There are different ways to write {es} queries and some of them are more 
+efficient than others. If you are operating on a big scale, make sure that your 
+{dfeed} query is as efficient as possible. Consult 
+{ref}/tune-for-search-speed.html[Tune for search speed] to learn more about {es} 
+performance tuning.
+
+
+[[influencers-detectors]]
+=== Choosing influencer fields and detectors
+
+Optimize your {anomaly-job} by choosing only relevant influencer fields and 
+detectors.
+
+
+[[short-bucket-span]]
+=== Avoiding short bucket span for large jobs
+
+If you have a large amount of data, use longer bucket span. If your data 
+requires short bucket span, split the data to avoid having high cardinality. The 
+combination of short bucket span and high cardinality is a resource-intensive 
+situation that you may want to avoid.
+
+
+[[population-analysis]]
+=== Using population analysis
+
+Population analysis is more memory efficient than individual analysis of each 
+series. It builds a profile of what a "typical" entity does over a specified 
+time period and then identifies when one is behaving abnormally compared to the 
+population. Use population analysis for analyzing high cardinality fields if you 
+expect that the entities of the population generally behave in the same way.
+
+
+[[pre-aggregating-data]]
+=== Pre-aggregating data
+
+{anomaly-jobs-cap} always work on summary statistics calculated for the buckets. 
+The statistics can be calculated in the job itself or via aggregations. It is 
+more efficient to use an aggregation when it's possible, as in this case, the 
+data node does the heavy-lifting instead of the {ml} node.
+
+In certain cases, you cannot do aggregations to increase performance. For 
+example, categorization jobs use the full log message to detect anomalies, so 
+this data cannot be aggregated. If you have many influencer fields, it may not 
+be beneficial to use an aggregation either, because only a few documents in each 
+bucket may have the combination of all the different influencer fields.
+
+Consult <<ml-configuring-aggregation>> to learn more.
+
+
+[[disabling-model-plot]]
+=== Disabling model plot
+
+Model plot calculates and stores the model bounds for each analyzed entity. It 
+generates a huge amount of results and has a significant impact on performance. 
+The indexing load is the same as for forecasting: one document per bucket per 
+split field value. If you have high cardinality fields and/or short bucket span, 
+disable model plot if it's not needed.
+
+
+[[results-retention]]
+=== Setting result retention time
+
+{anomaly-detect-cap} results are retained indefinitely by default. For big jobs, 
+results build up over time, and your result index will be quite large. Consider 
+how long you wish to retain the results and set `results_retention_days` 
+accordingly – for example, to 30 or 60 days – to avoid unnecessarily large 
+result indices. Deleting old results does not affect the model behavior.
```

Comment:
Is it worth cross referencing using a dedicated results index here?

Reply:
I don't think it is necessary to use an in-page link to emphasize one of the previous list items.